NodeMCU/src/Server/server.ino

```diff
@@ -0,0 +1,51 @@
+#include <ESP8266WiFi.h>
+#include <ESP8266WebServer.h>
+ 
+const char *ssid = "RonaZona";
+const char *password = "squidward420";
+
+bool activate = false;
+ 
+ESP8266WebServer server(80);
+
+void handleRequest(){
+  if(activate){
+    server.send(202, "", "");
```

Comment:
In the future, we should probably use variables that hold the proper response codes, for visual clarity.